frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -3714,7 +3758,8 @@ gatherAndFilterCandidatesForwarding(Context* context,
                                     const CallScopeInfo& inScopes,
                                     CandidatesAndForwardingInfo& nonPoiCandidates,
                                     CandidatesAndForwardingInfo& poiCandidates,
-                                    LastResortCandidateGroups& lrcGroups) {
+                                    LastResortCandidateGroups& lrcGroups,
+                                    std::vector<ApplicabilityResult>* rejected) {
```

Comment:
should we pass this `rejected` to the `filterCandidatesInstantiating` calls (for non-POI and POI) now that we have it?